### ISSUE TITLE
conversion to python 3

### ISSUE THREAD

**Issue by amapic** [experiencor/keras-yolo2#391]
Hi,
Is the code easy to adapt to python 3 ? I readed in former issues that there's changes to do.
Thank you.

**Comment by MBoaretto25** [comment]
Yes, However [here](https://github.com/rodrigo2019/keras_yolo2) is a version of this code that is already compatible with python3.
